src/MessagePack.UnityClient/Assets/Scripts/MessagePack/SequencePool.cs

```diff
@@ -76,7 +76,18 @@ public SequencePool(int maxSize)
         public SequencePool(int maxSize, ArrayPool<byte> arrayPool)
         {
             this.maxSize = maxSize;
-            this.arrayPool = arrayPool;
+            this.arrayPoolOrMemoryPool = arrayPool;
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SequencePool"/> class.
+        /// </summary>
+        /// <param name="maxSize">The maximum size to allow the pool to grow.</param>
+        /// <param name="memoryPool">Memory pool that will be used.</param>
+        public SequencePool(int maxSize, MemoryPool<byte> memoryPool)
```

Comment:
We never add new public APIs to `master`, which is our servicing branch.
I'll retarget your PR to `develop`.

Reply:
You also should address the compiler warnings. You can use "Quick Actions" in Visual Studio (or whatever the code fix activator is in your IDE) to have the changes to the PublicAPI.Unshipped.txt files automatically made.